package.json

```diff
@@ -38,7 +38,7 @@
   },
   "license": "SEE LICENSE IN LICENSE",
   "dependencies": {
-    "@defra/forms-model": "^3.0.264",
+    "@defra/forms-model": "^3.0.275",
```

Comment:
I've left https://github.com/DEFRA/forms-runner/pull/334 unmerged for now

Up to you if you'd rather merge then rebase this branch instead